week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,7 +21,13 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (num===Number || num % 2 ===0 && num<=100){
+    return true
+  } else { 
+    return false
+  }
+}
```

Comment:
In this function, you forgot to add typeOf num. the rest is correct

Reply:
done